tests/test_2022_10_25.py

```diff
@@ -0,0 +1,128 @@
+"""
+Tests for voting 25/10/2022.
+"""
+from scripts.vote_2022_10_25 import start_vote
+from utils.test.tx_tracing_helpers import *
+from utils.test.event_validators.payout import (
+    Payout,
+    validate_token_payout_event,
+    validate_agent_execute_ether_wrap_event,
+)
+from utils.finance import ZERO_ADDRESS
+from brownie.network.transaction import TransactionReceipt
+
+
+dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+lido_dao_token = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+weth_token = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+dai_token_address = "0x6B175474E89094C44Da98b954EedeAC495271d0F"
```

Comment:
Better either use dai fixture as `dai.address` or drop dai fixture & interface completely in favour for `interface.IERC20` provided by brownie

Reply:
fixed